pam/generate.sh

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env bash
 
 set -euo pipefail
+set -x
 
-PROJECT_ROOT=$PWD/..
+PROJECT_ROOT=$(realpath "$(dirname "$0")")/..
```

Comment:
Why is this better than using $PWD?

Reply:
`$PWD` is set properly only when this is called from the right working dir, which is fine in case of `go generate` but not when `pam/generate.sh` is called as a script. Now that's still broken because of other parts though, so I'll address better.